recipes-kernel/linux/linux-nilrt.inc

```diff
@@ -77,6 +77,10 @@ do_install_append() {
 	${WORKDIR}/export-kernel-headers.sh ${S} ${D}/kernel
 }
 
+do_shared_workdir_append() {
+   cp -rf * ${STAGING_KERNEL_BUILDDIR}
+}
+
```

Comment:
Nits:
1. It looks like you're using soft-tabs here, where the rest of the recipe uses hard-tabs. Could you make it consistent?
2. As long as you're touching up the whitespace, I would recommend using `./*` for the wildcard - only because it seems more clear to me.